exercises/linked-list/example.js

```diff
@@ -14,11 +14,12 @@ export default class Deque {
       return undefined;
     }
 
-    const value = this.head.value;
+    const { value } = this.head;
     if (this.head.next) {
       this.head = this.head.next;
       this.head.prev = null;
     } else {
+      // eslint-disable-next-line no-multi-assign
```

Comment:
please don't disable properties which can be easily fixed.

Reply:
Agreed.